dipy/reconst/csdeconv.py

```diff
@@ -671,6 +675,11 @@ def odf_deconv(odf_sh, R, B_reg, lambda_=1., tau=0.1, r2_term=False):
             # SVD did not converge in Linear Least Squares in current
             # voxel. Proceeding with initial SH estimate for this voxel.
             pass
+        except ValueError as vae:
+            # SVD did not converge in Linear Least Squares in current
+            # voxel. Proceeding with initial SH estimate for this voxel.
+	          warnings.warn('ValueError: Computation of fODF did not converge')
+            pass
```

Comment:
Indentation in this block seems to be off (`pass` doesn't match any indentation level). See errors here: https://travis-ci.org/nipy/dipy/jobs/209265093#L2490